server/lib/mappings/result-data-sample.json

```diff
@@ -73,7 +73,12 @@
                     "type": "long"
                 },
                 "measurement_title": {
-                    "type": "text"
+                    "type": "text",
+                    "fields": {
+                        "raw": {
```

Comment:
Perhaps `raw` indicates something besides a `keyword`?  What about changing the field to be a `keyword` and adding a `text` sub-field which is of type `text`?

Reply:
We need a `keyword` type to sort/aggregate, and this is used for aggregation. Now, whether we really _also_ need to support full text search on this is a separate question. The search query is a bit more free-form, but "title" just suggests something that should be text searchable to me.

Which field is on the top really doesn't make a difference, logically.